src/iotjs_binding.c

```diff
@@ -85,6 +86,44 @@ double iotjs_jval_as_number(jerry_value_t jval) {
 }
 
 
+bool iotjs_jval_get_data(jerry_value_t jval, iotjs_string_t* getbuffer) {
+  iotjs_bufferwrap_t* buffer_wrap = iotjs_jbuffer_is_bufferwrap(jval);
```

Comment:
We should only do this check if value is not string.

Reply:
I find the function name a bit strange because it is not a boolean function. `iotjs_jbuffer_get_bufferwrap_or_null`